lib/modulesync/repository.rb

```diff
@@ -158,7 +158,7 @@ def submit_changes(files, options)
           repo.push('origin', branch, opts_push)
           puts "Changes have been pushed to: '#{branch}'"
         end
-      rescue Git::GitExecuteError => e
+      rescue Git::Error => e
```

Comment:
I thought I could solve this deprecation warning:
```
DEPRECATION WARNING: Git::GitExecuteError is deprecated! Use Git::Error instead. (called from rescue in submit_changes at /home/runner/work/modulesync/modulesync/lib/modulesync/repository.rb:161)
```

But now: `uninitialized constant Git::Error`.